src/schemas/components/SchemaFormPropertyInput.vue

```diff
@@ -67,14 +81,16 @@
     }
 
     if (isSchemaPropertyType(type, 'array')) {
-      // new list input from prefect-design
-      throw 'not implemented'
+      return withProps(SchemaFormPropertyArray, {
+        property: { ...property.value, type },
```

Comment:
This spread technically isn't necessary. But it helps typescript narrow the type for whole property.